recipe/meta.yaml

```diff
@@ -45,14 +42,13 @@ requirements:
     - thrift-cpp
   host:
     - abseil-cpp 20211102.0  # [not osx]
-    - aws-sdk-cpp 1.8.185
+    - aws-sdk-cpp 1.10.55
+    - aws-crt-cpp 0.18.16
     - boost-cpp 1.82
     - brotli 1.0.9
     - bzip2 1.0.8
     # Only required and used on Windows.
-    - c-ares 1.15.0  # [win]
-    # The binary using this presumably isn't built on Windows.
-    - gflags 2.2.2  # [not win]
+    - c-ares 1.18.1  # [win]
```

Comment:
Could it be built with **c-ares 1.19.1**? Because versions `<1.19.1` are affected by CVE-2023-32067

Reply:
No need. c-ares has a run exports on the major. At runtime 1.19.1 will be picked by the solver.